web/wp-content/themes/dctx-v2/inc/template-tags.php

```diff
@@ -0,0 +1,540 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-atl3ea.php, but found phpcs-scan-aTL3Ea.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).